common/djangoapps/student/signals/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .signals import (
```

Comment:
I thought we were staying away from relative imports? Not sure if it matters as much since this is in the __init__ module?

Reply:
I'm fine with changing this to an absolute import.I defaulted to a relative import because it's clear that the code is importing something for *this* Django app instead of outside the app. Is the absolute import decision something that we've codified somewhere?